src/core/linalg/src/sparse/4C_linalg_graph.cpp

```diff
@@ -57,28 +56,92 @@ Core::LinAlg::Graph& Core::LinAlg::Graph::operator=(const Graph& other)
   return *this;
 }
 
-int Core::LinAlg::Graph::insert_global_indices(int GlobalRow, int NumIndices, int* Indices)
+void Core::LinAlg::Graph::fill_complete()
 {
-  return graph_->InsertGlobalIndices(GlobalRow, NumIndices, Indices);
+  int err = 0;
+
+  if (graphtype_ == CRS_GRAPH)
+    err = graph_->FillComplete();
+  else if (graphtype_ == FE_GRAPH)
+    err = static_cast<Epetra_FECrsGraph*>(graph_.get())->GlobalAssemble();
+
+  if (err != 0) FOUR_C_THROW("Graph::fill_complete() returned {}.", err);
 }
 
-int Core::LinAlg::Graph::insert_global_indices(
-    int numRows, const int* rows, int numCols, const int* cols)
+void Core::LinAlg::Graph::fill_complete(const Map& domain_map, const Map& range_map)
 {
   int err = 0;
 
   if (graphtype_ == CRS_GRAPH)
-    FOUR_C_THROW("This type of insert_global_indices() only available for FE_GRAPH type.");
+    err = graph_->FillComplete(domain_map.get_epetra_block_map(), range_map.get_epetra_block_map());
   else if (graphtype_ == FE_GRAPH)
     err = static_cast<Epetra_FECrsGraph*>(graph_.get())
-              ->InsertGlobalIndices(numRows, rows, numCols, cols);
+              ->GlobalAssemble(domain_map.get_epetra_map(), range_map.get_epetra_map());
 
-  return err;
+  if (err != 0) FOUR_C_THROW("Graph::fill_complete() returned {}.", err);
```

Comment:
Technically, it does not return. Rather an error occurred.

This comment also applies to the other methods in the wrapper.

Reply:
The name of the function is already reported in the stack trace.